### DISCUSSION TITLE
Coinbase Learning Rewards & Quests Proposal

### DISCUSSION THREAD

**Discussion by lechenghiskhan** [akash-network/community#513] [Governance Proposals]
**Overview**

Over the past year and a half, Akash has seen major growth in network activity and functionality. The community of developers and users is steadily increasing over time. With the Coinbase [announcement](https://twitter.com/CoinbaseAssets/status/1769753664904655039) of listing AKT as a supported asset, this is the perfect opportunity to expand the network’s user base through education on a Tier 1 platform.
\
\
**Why Coinbase Learning rewards?**

[Coinbase Learning Rewards](https://www.coinbase.com/learning-rewards) is an educational platform within Coinbase that allows Coinbase users to learn about projects across the crypto ecosystem. The program provides a small monetary reward in the form of tokens to those that complete lessons and quests. 
\
\
**Budget & Outcomes**

The Akash core team will run 2-3 separate Learn campaigns with Coinbase to educate new users about Akash Network and foster interest in the work being done across the community. For these campaigns, we request $8,000,000 or ~1,612,903 AKT using a 30-day moving average price of ~$4.96 (as of block height 15,488,114). The Community Pool currently holds 4,532,193 AKT and is refilling at a rate of 1.156 million AKT per month. 

Any unused funds will be returned to the Community Pool.

With an $8M budget and expected total payout of $10-15 per user for lessons and quests, exhausting the total budget would increase holders of AKT by 533,333 - 800,000. For reference, according to Smart Stake Analytics there are currently ~101,000 [Active Addresses](https://analytics.smartstake.io/akash/richlist/address) on-chain. 
\
\
**Timeline**

Given the urgency of the timeline, the discussion period will be shortened to expedite the process and maintain momentum behind this incredible opportunity. 

Github Discussion: Open for three days of review and feedback

Governance Proposal live for voting: Friday, March 22, 2024

Coinbase Learn campaign estimated go live: End of March 2024
\
\
Voting:

* By voting YES to this proposal, you agree:
    * Approval for the full funding amount
    * Authorize the community pool to disburse AKT (1,612,903) for Coinbase Learning Rewards to a publicly visible wallet. 
* By voting NO, you disagree that community pool resources should be used for Coinbase Learning Rewards.
* By voting ABSTAIN, you express no opinion on the matter.

AKT Wallet Address: TBD

**Comment by ghost** [comment]
Coinbase learning is a great platform to learn basics about general crypto and specific projects, I used it myself a lot in my first months of my crypto journey 

I support this proposal

**Comment by tombeynon** [comment]
Full support for this - exposing new users to Akash and educating them that this is so much more than just a token is the most important thing we can achieve right now. Given the rate the community pool is refilling, 1.5 months worth of funds is a very reasonable price to pay for potentially more than doubling the active accounts on-chain. 

ECO Stake would definitely vote yes for this proposal.

**Comment by rianfloo** [comment]
The proposal to allocate a budget for Coinbase Learning presents an excellent opportunity for enhancing the visibility of AKT.

By the way, I have a few questions:

- Could you explain the methodology behind the projected increase in AKT holders, ranging from 533,333 to 800,000? Is this estimate based solely on the rewards per user anticipated following the Learning campaign?

- Does the proposed reward of $10 to $15 per participant seem substantial in comparison to rewards offered for other cryptocurrencies featured in the program? In other words, is the strategy aimed at offering more attractive rewards to motivate users to engage in quests?

- Regarding the quests that involve answering questions and completing various actions, could you provide more details on the types of quests that will be available?

- I recall Fetch.ai implementing a similar strategy on Coinbase approximately two to three years ago. Are there any available data or insights regarding the outcomes of past Coinbase Learning campaigns?

**Reply by lechenghiskhan** [reply]
Thank you for your input @rianfloo 

1. Yes. This is an estimate of how many new AKT holders the learn campaign would add to the network given the approximate cost of $10-15 (roughly) for users who complete the learn lessons and quests.

2. This reward range is the average reward typical of this program for completing a learn campaign.

3. There are two broad types: lessons (which are educational followed by simple quizzes) and quests (which are more substantial). We plan on running several campaigns that are a combination of both over (what is likely to be) several months. For the initial campaign, the OCL core team proposes a learn-heavy campaign. These would include educating users on what Akash is (supercloud), what Akash is capable of running (everything), and much more. 

4. This is not data we have access to. Based on the information available, the learn campaigns designed will be structured in a way that appeals to the majority of Coinbase's retail users.

**Reply by rianfloo** [reply]
Thanks mate. We need communication / "hype". Let's do it.

**Comment by Aihonaken** [comment]
So, atm it's controversial for me. Despite i'm always advocating that retail investors are the major fuel. But, 
1: that's a lot of money ! We spent less on provider incentivze. 
2. How are you going to determine if we reach the goal, cause afaik the AKT will be sent to the Coinbase wallets. There is nothing in COinbase learning that will force people to create new wallets (correct me if i'm wrong)
3. What we are going to do if the goal is not going to be reach
4. I would rather give 4mil to increase the $AKT volume imo
5.  Who's in charge of the videos and Quiz? 
6. Again, how are we going to determine the success ( # of wallets is not an option, since it can be created by any bot) 
7. Again this is a lot of money, is there a chance to reduce the amount?

**Reply by lechenghiskhan** [reply]
Thanks @Aihonaken for your input, as always. 

1. It is substantial and it has to be. There is one opportunity to launch and that opportunity has to be seized. This number was arrived at after substantial discussion and deliberation given the data and information available. Furthermore, the provider incentive program was just a pilot. If the pilot proves successful, the total incentives unlocked in the future will be greater.

2. In order to complete the lessons/quests, users will have to have AKT wallets set up to receive the rewards.

3. The entire budget will not be used in a single campaign, but deployed over multiple campaigns. This phased approach will allow us to learn from the initial campaign before launching subsequent campaigns.

4. As previously stated, this number was determined after substantial deliberation and discussion

5. The Overclock Labs Core Team is working directly with the Coinbase Learn team to design and build these lessons, quests, and quizzes.

6. High level, the number of completions (which will roughly equate to the number of new holders of AKT). All of CB's users are KYC'd and I don't think bot wallet creations under CB registered accounts are possible w/o violating their TOS. 

7. No, there is not. Again, this budget isn't something that will be exhausted in a single shot, but deployed across several campaigns.

**Reply by Aihonaken** [reply]
Thank you for the answers @lechenghiskhan . i've been thinking on this prop, and despite the fact that i don't like to give so much money to learning incentives, i also might be unaware of a full potential of coinbase. The market is in a good condition rn, so I guess this could be the opportunity we are looking to spread awareness. Let's give it a shot, gladly support the prop!

**Comment by SasoLithops** [comment]
Hi!

Long time Akash holder and staker here.

May I know how you got to numbers 533,333 and 800,000? Is 8M$ result of these numbers as reward between 15$-10$?

Or are numbers just predictions on past participation on Coinbase Learning Rewards?
And as you said any unused funds will be returned to Community Pool?

Due to high ammount it would be good to get data how many of these participants actually show interest and stay?

**Reply by lechenghiskhan** [reply]
Thank you @SasoLithops! 

1. That is exactly it; this is an estimate based on the average cost per completion, which is derived from prior Learn campaigns as guided by the Coinbase team.

2. As far as user behavior post-campaign, any data that The OCL core team is allowed to share will be.

**Reply by SasoLithops** [reply]
Thank you @lechenghiskhan for all replies you posted. Makes more sense to me now. It was a huge ammount but I know listing on Coinbase is huge too.

**Comment by PavloDereniuk** [comment]
Hi!

Where are these figures sourced from? The proposal lacks clarity and specifics. It's essential to elucidate the rationale behind the request for 8 million. Do you possess any supporting research or additional information?

Given the magnitude of the proposal - 8 million - it's concerning that there's insufficient detail provided to evaluate its potential impact on the project and the community.

**Reply by lechenghiskhan** [reply]
Thanks @PavloDereniuk for your input here. 

As answered previously, the $8mm number was arrived at based on lengthy discussions, historical campaign sizes, and various other factors. Ultimately, AKT's listing on Coinbase is an extraordinary event and the learn campaign is an opportunity that should be seized to further educate (or elucidate 😄) new users/holders about Akash Network and the role we play in AI and technology in general.

The $8mm budget will likely be used across several (2-4) campaigns with each campaign running for 2-3 weeks. For reference, AKT's 24-hour trading volume over the last day was ~$18mm, so $8mm over 4 campaigns spanning a total of 8-12 weeks will have a relatively low impact on float.

**Reply by PavloDereniuk** [reply]
Thank you for your detailed response. After reading the previous answers as well, this idea seems more logical to me. I'm hopeful that it will succeed.

**Comment by lolobu12** [comment]
Who says that there is not a huge intersection between Coinbase users and already educated AKT holders which, like me, will surely do this quiz as well to grab the free money. 
The amount of people being confronted with AKT for the first time seems not as high as you show.

Another reason to question this number is the high exposure of AI/dePIN in crypto news which always has AKT mentioned all over
Greg is very outspoken in social media as well which leads to a very active Akash presentation.
All in all I doubt the intersection of new users with that compaign. 

Akash is a top100 coin in a narrative which is hot already. 8M seems a lot to me having doubts how many "new" people this will attract.

**Reply by lechenghiskhan** [reply]
Hi @lolobu12 I appreciate your response and the concern is noted.

Nonetheless, this is a unique and extremely rare opportunity to expand the Akash footprint.
Beyond the top 100, Akash Network's ultimate success metric is mainstream adoption. 
Collectively, we need to reset our sights higher.

**Reply by lolobu12** [reply]
Hi, you don't refer to my arguments that there is much less interection to adopt new users.
Top 100 coins are already well known and mainstream adoption in terms of Akash usage is not retail token holders but contract deployment and GPU usage.

The latest investment into GPUs and presentations on KubeCon etc. seem more in line with Akash expension than a educational quiz. 

Maybe I am biased as I cannot put myself into shoes of somebody new to crypto and therefore a quiz might actually offer a real new insight into such coin, but for 8M you can even promote an akash owned channel with educational content, shows, live deployment, series like to follow up on KubeCoin. 
8M and a questionalbe user base on Conbase is just not sitting well with me. 
"Akash Network's ultimate success metric is mainstream adoption", so you think retail token purchase on Cex is the ultimate goal and not buy/sell computing resource with token back up due to deployment on Akash consule?

**Reply by lechenghiskhan** [reply]
Thanks, @lolobu12 

There is no guarantee exactly how many new users this campaign can bring to Akash Network. The numbers presented are estimates, after all. 

You're also offering objections with no concrete alternatives. I want to point you to [this](https://github.com/orgs/akash-network/discussions/448#discussioncomment-8192276) as a model example of an incredibly well-thought-out and constructive piece of feedback from a community member on a previous discussion.

When we're thinking about education and utility for AKT, there are several perspectives to consider:
1. Utility of Akash as the Supercloud. In this vein, the provider incentives serve this purpose beautifully.
2. Technical growth and development. Hackathons, Akash EDU, bounties (Zealy), and other funded initiatives fulfill this role.
3. Support. Vanguards and insiders have done wonders here.
4. Awareness, presence, and marketing: This is where Coinbase Learn comes in. The opportunity to reach a new audience (even if that audience is partial) at the scale of millions of CB users is a rare opportunity. This will be the biggest awareness, education, and distribution campaign Akash Network has ever done. 

If you're arguing that most users on Coinbase already know of or hold AKT, then why should Akash Network list on any new exchanges at all? Reaching new users and increasing access is what tier 1 exchanges offer. Just two days after listing, volume on CB has already exceeded Kucoin as the only other AKT/USD pair.

**Comment by rodri-r** [comment]
Fully support this, it should help grow the network in several ways!

**Comment by piber-dev** [comment]
hello @lechenghiskhan,

although the budget for this proposal is massive, i totally support this proposal. from my understanding, a similar amount (+- depending on a number of factors) will be spent **per user** here as on zealy. the key difference is that this proposal has a huge outreach and a huge budget, which i think can help make the network grow immensely. i truly believe this opportunity is great and should be seized.

**Comment by nick134-bit** [comment]
Hey @lechenghiskhan, does coinbase charge any fees for this service? Like if we allocate 1,612,903 AKT an x amount is going directly to coinbase as fee. Or per user / completion.
Also are only user with a verified status on coinbase allowed to participate?

**Reply by lechenghiskhan** [reply]
Thanks @nick134-bit 

There is a transaction fee per completed quest that CB charges. This effectively pays for all of the overhead like integration, support, etc.

I'm not at liberty to share the exact information, but I can say that it's a fraction of each dollar rewarded.

**Reply by nick134-bit** [reply]
Sounds like a great deal!

**Comment by brewsterdrinkwater** [comment]
The Coinbase Learning Rewards & Quests Proposal is [on chain for a vote. ](https://www.mintscan.io/akash/proposals/252)

**Comment by Zblocker64** [comment]
I may be late to the party, but I am in support of this proposal. Yes it’s a lot of AKT, but the result will be more AKT returning to the community pool through network usage. Aurora Staking will be voting yes!

**Comment by SophoclesSummer** [comment]
This proposal was approved on 3/29/2024 by a 81% majority. 
https://www.mintscan.io/akash/proposals/252
